src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
```

Comment:
Avoid passing expressions directly into the constructor. It's better to create a variable for the 'side' parameter and then pass it to the constructor for better code readability.